framework/src/wifi_manager/wifi_manager.c

```diff
@@ -528,8 +596,19 @@ wifi_manager_result_e wifi_manager_set_mode(wifi_manager_mode_e mode, wifi_manag
 		return WIFI_MANAGER_SUCCESS;
```

Comment:
how about to protect g_manager_info ?

Reply:
Yes. I include this in the critical section.